components/tabs/tab.js

```diff
@@ -0,0 +1,47 @@
+import { css, html, LitElement, unsafeCSS } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+import { getFocusPseudoClass } from '../../helpers/focus.js';
+import { RtlMixin } from '../../mixins/rtl/rtl-mixin.js';
+import { SkeletonMixin } from '../skeleton/skeleton-mixin.js';
+import { TabMixin } from './tab-mixin.js';
+
+class Tab extends TabMixin(SkeletonMixin(RtlMixin(LitElement))) {
```

Comment:
I don't think `RtlMixin` is needed here, since we're using the CSS logical properties instead. This means we can remove the corresponding import above too.